runtime/runtime/config/config.go

```diff
@@ -162,9 +162,10 @@ type PubsubTopic struct {
 }
 
 type PubsubSubscription struct {
+	ServerID   int    `json:"server_id"`   // the index into (*Runtime).PubsubServers
```

Comment:
Does this indirection really make sense now? Or would it make more sense to just put the project id and service account in here? (Happy to keep it as-is if we don't want to make changes at this point)